testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/multipart/ContextProvidersTest.java

```diff
@@ -194,7 +194,7 @@ void doTestGetList() throws Exception {
             InputPart inputPart = it.next();
             customers.add(inputPart.getBody(ContextProvidersCustomer.class, null).getName());
          }
-         Assert.assertThat("Wrong count of customers from response", new Integer(customers.size()), is(2));
+         Assert.assertThat("Wrong count of customers from response", Integer.valueOf(customers.size()), is(2));
```

Comment:
This is minor and applies to the rest of the changes in this file, but we shouldn't need to wrap these in an object. Also `Assert.assertThat` is deprecated as well so IMO we could switch to `Assert.assertEquals("Wrong count of customers from response", 2, customers.size())`.